### ISSUE TITLE
Distribute Affine for Linux as something not AppImage​

### ISSUE THREAD

**Issue by justin13888** [toeverything/AFFiNE#5985]
### Description

While I'm glad that Affine supports a desktop application for Linux, there is still some room for improvement to make it a more comfortable experience for desktop Linux users.

Currently, Affine for Linux is distributed as an AppImage.

A few obvious drawbacks without exhaustively elaborating too much include:
- Integration with desktop environment either requires manual configuration or tool like AppImageLauncher. Icon also wouldn't be configured automatically
- No integration with display server (i.e. Wayland, X.org) configurations. For Wayland desktops, you have to manually pass Ozone flags for Electron (used by Affine) to force it to render with Wayland.
- Lack of dependency management: This means large binary sizes, slow dependency updates (e.g. Electron version)

While there are many Linux package formats, I understand Linux desktop users is not a particularly large userbase and regardless, the chosen distribution method(s) should require the least amount of active maintenance effort.

I would suggest distributing as a Flatpak and if possible, additionally through FlatHub. As far as I know, this is one of the most streamlined option to bring Affine to as many desktop Linux users as possible. FlatHub is also becoming a centralized marketplace for Linux desktop applications (imagine Google Play Store for Linux)! Having improved Linux support will also be another clear win over Notion itself.

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by pengx17** [comment]
Thanks for the suggestion! We are working on providing more distribution formats right now and finding solutions for auto update for linux bundles.

**Comment by pengx17** [comment]
Auto updating for AppImage should now be supported. We are investigating flatpak bundling right now but it may not happen very soon. In the meantime, help wanted!

**Comment by pengx17** [comment]
Most issues related to AppImage should have been fixed lately (when using AppImageLauncher):
- icons in the file explorer
- icons on the Dock
- Wayland support

**Comment by zuhairamahdi** [comment]
is there any update for this issue?

**Comment by pengx17** [comment]
> @zuhairamahdi is there any update for this issue?

Yes, the first version of deb/flatpak distribution is wip https://github.com/toeverything/AFFiNE/pull/8439
